xarray_jax/tests/xarray_jax_test.py

```diff
@@ -68,7 +52,7 @@ def ops_via_xarray(inputs):
                                data=inputs,
                                coords={'lat': np.arange(3) * 10,
                                        'lon': np.arange(4) * 10})
-      x = np.abs((x + 2) * (x - 3))
+      x = abs((x + 2) * (x - 3))
```

Comment:
`abs` works although it's causing a pytype error on our side for some reason. Could you replace with `xu.abs` (`import xarray.ufuncs as xu`)
which also works, doesn't trigger pytype and will be the recommended way to call ufuncs once the wrapper is no longer present.